appveyor.yml

```diff
@@ -0,0 +1,12 @@
+version: 0.4.{build}
```

Comment:
It's overwritten. When the Appveyor build starts, it initially displays this but quickly changes to our defined version. It comes with the default appveyor.yml if you have AppVeyor create it for you. Could be removed, I imagine.